examples/object_detection/ios/RunScripts/download_models.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+set -ex
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+MOBILENETV1_SSD_URL="https://storage.googleapis.com/download.tensorflow.org/models/tflite/task_library/object_detection/ios/lite-model_ssd_mobilenet_v1_1_metadata_2.tflite"
```

Comment:
Use these:
https://storage.googleapis.com/ai-edge/interpreter-samples/object_detection/ios/efficientdet_lite0.tflite
https://storage.googleapis.com/ai-edge/interpreter-samples/object_detection/ios/efficientdet_lite2.tflite

Reply:
We don't use MOBILENETV1_SSD and EFFICIENTDET_LITE2 right?

Reply:
@PaulTR 
Their models return 76824 boxes and we need to calculate IOU to remove boxes overlap, then the task will be a little late, sorry about that

Reply:
@PaulTR 
I make a demo in colab using new model, but I don't know how to map box output correctly. (origin, scale).
Please help me check.
Thanks
https://colab.research.google.com/drive/10ShNFLsSGWZPMvFng4HPzDuC3lStcGXH?usp=sharing

Reply:
If we're not using all of these models, then we should remove the unused ones.